svgdigitizer/entrypoint.py

```diff
@@ -571,6 +594,32 @@ def _create_linked_svg(svg, png, svg_template):
         drawing.save(pretty=True)
 
 
+@click.command()
+@click.option(
+    "--template",
+    type=str,
+    default=None,
+    help="Add template elements in svg files. Options: basic, file:<file path>",
+)
+@click.option(
+    "--outdir",
+    type=click.Path(file_okay=False),
+    default=None,
+    help="Write output files to this directory.",
+)
+@click.argument("img")
+def create_svg(img, template, outdir):
+    r"""
+    Write an SVG that shows `png` or `jpeg` as a linked image.
```

Comment:
so we would nee tests for png and jpeg and the help method should indicate which image formats are supported. TIFF, bmp, raw, etc